examples/lock-app/efr32/README.md

```diff
@@ -65,10 +65,15 @@ Silicon Labs platform.
         915MHz@19dBm
     -   BRD4304A / SLWSTK6000B / MGM12P Module / 2.4GHz@19dBm
 
```

Comment:
I don't understand what you mean here by factorize and include link? Can you explain ? I might be able to do that in a furtur PR if neeeded

Reply:
All commissioning is the same so far for all examples using EFR32 , instead of copy pasting the README in each examples you could use a single readme files for all .